includes/components/class-review.php

```diff
@@ -251,9 +254,35 @@ public function alter_listing_manage_menu( $items, $menu ) {
 			$listing = $menu->get_context( 'listing' );
 
 			if ( $listing && $listing->get_rating_count() ) {
+
+				// Set query arguments.
+				$query_args = [
```

Comment:
if possible, use our models API, you can use a simple string key like "review_count" for cache instead of array of query arguments in this case, it's already pretty unique